demo/molecule/photoUploader/playground

```diff
@@ -0,0 +1,147 @@
+const demoStyles = {
+  margin:'auto',
+  marginTop: 20
+}
+
+const lineStyles = {
+  color: 'black',
+  paddingLeft: 20
+}
+
+const _maxPhotos = 10
+const _mainPhotoLabel = 'PRINCIPAL'
+const _addPhotoTextSkeleton = 'Añadir más'
+const _dropPhotosHere = 'Suelta las fotos aquí'
+const _addPhotoTextButton = 'Seleccionalas de tu dispositivo'
+const _dragPhotoTextInitialContent = 'Arrastra las fotos aquí'
+const _limitPhotosUploaded = 'Ha alcanzado el límite de fotografías'
+const _limitPhotosUploadedNotification = 'Máximo de fotografias alcanzado'
+const _errorFormatPhotoUploaded = 'Las fotografías deben tener formato JPEG, PNG, TIF o GIF'
```

Comment:
TIF format?
Then the server serve a JPG or PNG format?

Reply:
We were accepting by default all MIME image types, but now we change it to only accept by default these MIME type fles:
`image/jpeg, image/gif, image/png, image/webp, image/bmp`
These are tested and working fine, and maybe more are compatible with no iteration of the component. For now, TIFF files fails when uploading 😢 but it's not a problem with UX requirements. 

The component transform the image to JPEG, and compress it a little. I'm gonna explain better how the component works in README, and in the PR description.